zerver/lib/events.py

```diff
@@ -216,6 +216,10 @@ def fetch_initial_state_data(user_profile: UserProfile,
         state['realm_push_notifications_enabled'] = push_notifications_enabled()
         state['realm_upload_quota'] = realm.upload_quota_bytes()
         state['realm_plan_type'] = realm.plan_type
+        state['plan_includes_wide_organization_logo'] = realm.plan_type != Realm.LIMITED
+        state['upgrade_text_for_wide_organization_logo'] = "Available on all paid plans. Upgrade "
```

Comment:
let's make this string a constant UPGRADE_TEXT_STANDARD.
Also, `Upgrade` should be a link to /plans.

Reply:
> let's make this string a constant UPGRADE_TEXT_STANDARD, and also mark the whole string for translation.

Right, I've made this variable as a member of the Realm class.

> Not sure the best way to make Upgrade a link to /plans. Doing it in the template as it is currently isn't ideal; maybe Tim will have ideas

Agreed, I wasn't completely sure of what we wanted this string to look like with the hyperlink (because we're marking the string for translation, I don't think including HTML in it would be pretty, so I didn't implement it the way you mentioned it in the issue).

> Also, I see in the html you've linked to /upgrade. I should have specified that the link should be to /plans

Change made.